src/usr/local/buildpack/tools/bundler.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+check_command ruby
+
+. /usr/local/buildpack/utils/ruby.sh
+
+gem_install
+gem_shell_wrapper
```

Comment:
```suggestion
gem_shell_wrapper bundler
```

?

I see a difference between this and `pod` below

Reply:
Yes, the arg is only required when different to tool name. 😉